rpcs3/Emu/System.cpp

```diff
@@ -1598,8 +1599,12 @@ game_boot_result Emulator::Load(const std::string& title_id, bool is_disc_patch,
 				return game_boot_result::invalid_bdvd_folder;
 			}
 
+			// TODO: Check this
 			vfs::mount("/dev_bdvd", bdvd_dir);
```

Comment:
We already checked this. It's what was needed for wkc or whatever

Reply:
I mean what is the actual path used.

Reply:
It doesn't matter, as there is no disc, so you can't read or write any files to the folder

Reply:
And the virtualization is oblivious to the game

Reply:
It matters for timestamps obtainining and other sys_fs operations.

Reply:
Clarified.